open-craft/edx-platform
# [BB-2779] Cherry pick all upstream security fixes

This PR cherry-picks all the security fixes/changes made to the upstream `open-release/juniper.master` branch after the `open-release/juniper.2` tag was cut, that are not already in our common branch. This also includes a fix for profile image storage on S3, Django version upgrade and a Studio frontend version bump.

This PR also reverts a custom commit made by @pkulkark in https://github.com/open-craft/edx-platform/pull/237 and a fix related to that and then adds back the corresponding upstream commit and the same fix.

**Reviewers**:

**Testing instructions**:
* Please check the commits against the ones in the upstream `open-release/juniper.master` branch and verify that everything is okay.
* Verify that the fix made by @pkulkark is intact.
* Optionally, spawn a sandbox to test and verify that nothing is broken and everything works okay.
